rn-test.config.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+module.exports = {
+  require: require.resolve('./rn-test.require.js'),
+  runner: 'zora',
+  modules: [
+    'react-native-url-polyfill'
+  ]
```

Comment:
can you explain this ?

Reply:
React Native's URL implementation provides little functionality. Without the polyfill, the tests won't pass.

Reply:
why isnt this in the other file with the other polyfills ?

Reply:
What file? The polyfill is injected in the require script.